src/test/java/apoc/export/json/ExportJsonTest.java

```diff
@@ -0,0 +1,260 @@
+package apoc.export.json;
+
+import apoc.graph.Graphs;
+import apoc.util.TestUtil;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Scanner;
+
+import static apoc.util.MapUtil.map;
+import static org.junit.Assert.assertEquals;
+
+public class ExportJsonTest {
+
+    private static final String EXPECTED_QUERY_NODES = String.format("[ {\n" +
+            "  \"id\" : 0,\n" +
+            "  \"labels\" : [ \"User\" ],\n" +
+            "  \"properties\" : {\n" +
+            "    \"name\" : \"foo\",\n" +
+            "    \"age\" : \"42\",\n" +
+            "    \"male\" : \"true\",\n" +
+            "    \"kids\" : [ \"a\", \"b\", \"c\" ]\n" +
+            "  },\n" +
+            "  \"relationships\" : [ \"(0)-[KNOWS,0]->(1)\" ]\n" +
```

Comment:
I don't want to see this format. anywhere, it's opaque. Please *do not* use `relationship.toString()`